### ISSUE TITLE
Nuclear-expansion like approach

### ISSUE THREAD

**Issue by maximilian-heeg** [kharchenkolab/Baysor#101]
Hi,

I was wondering if it is possible to tweak to Baysor parameters to get a nuclear expansion like approach. Basically, for our dataset, we have an extremely good DAPI segmentation, but only approximately 30% of the transcripts fall into the nucleus. 
However, running Baysor creates a lot more cells than expected, even if the prior-segmentation-confidence it set to `1`. I have tried to change `min_molecules_per_segment`, but this did not seem to do the trick.

Would there be an easy way to do this?

Thanks again,
Max

**Comment by VPetukhov** [comment]
Apologies for the late reply. I think the simplest way would be to remove all cells without nuclei if you think they aren't real. But if there is actually only 30% of transcripts in DAPI, there is a good chance that some DAPI aren't captured. So cells without DAPI could also be real. Are there evidence that DAPI expansion is a better option here? I can add it easily.

**Comment by maximilian-heeg** [comment]
Not really. The reason I was asking is, that Baysor always seemed to overestimate the number of cells in my samples. But with the new 10x multimodal segmentation and a high prior confidence, this got significantly better for me.
